drivers/pwm/pwm_nxp_flexio.c

```diff
@@ -101,6 +101,8 @@ static int pwm_nxp_flexio_set_cycles(const struct device *dev,
 	FLEXIO_Type *flexio_base = (FLEXIO_Type *)(config->flexio_base);
 	struct nxp_flexio_child *child = (struct nxp_flexio_child *)(config->child);
 	enum pwm_nxp_flexio_polarity polarity;
+	uint32_t duty_cycle = period_cycles - pulse_cycles;
+	bool is_only_low_or_high = false;
```

Comment:
variable name not the greatest here either because it's not specific about what is only low or high, I do get it, but prefer to call it like "do_no_modulation" or similar, but this is up to you

Reply:
is_signal_modulated? I think that might be a good middle ground.